src/fu-tool.c

```diff
@@ -3775,6 +3864,21 @@ main(int argc, char *argv[])
 			      /* TRANSLATORS: command description */
 			      _("Erase all firmware update history"),
 			      fu_util_clear_history);
+	fu_util_cmd_array_add(cmd_array,
+			      "get-bios-attribute,get-bios-attributes,"
+			      "get-bios-settings,get-bios-setting",
```

Comment:
eigh? I think we want to pick one, no?

Reply:
I'm torn on this.  Since it's attributes API in the kernel I wanted that in there. But people usually think of it as settings.

So it seemed no harm to just use aliases

Reply:
I think the harm is user confusion. Maybe one alias, but not 4...

Reply:
But if you think there "should be only one" LMK which and I'll tear out the others.

Reply:
I guess I lean on "get-bios-settings" with a single alias to "get-bios-setting".

Reply:
Is it returning one thing or multiple things?

Reply:
no arguments -> all the settings
one argument -> one setting
two arguments -> two settings
etc

Reply:
OK went for one with one alias.